echopype/calibrate/api.py

```diff
@@ -34,11 +36,101 @@ def _compute_cal(
         return cal_obj.compute_Sp(waveform_mode=waveform_mode, encode_mode=encode_mode)
 
 
-def compute_Sv(echodata: EchoData, **kwargs):
-    """Compute volume backscattering strength (Sv) from raw data."""
+def compute_Sv(echodata: EchoData, **kwargs) -> xr.Dataset:
+    """
+    Compute volume backscattering strength (Sv) from raw data.
+
+    The calibration routine varies depending on the sonar type.
+    Currently this operation is supported for the following ``sonar_model``:
+    EK60, AZFP, EK80 (see below for detail).
+
+    The EK80 echosounder can be configured to transmit
+    either broadband (``waveform_mode="BB"``) or narrowband (``waveform_mode="CW"``) signals.
+    When transmitting in broadband mode, the returned echoes are
+    encoded as complex samples (``encode_mode="complex"``).
+    When transmitting in narrowband mode, the returned echoes can be encoded
+    either as complex samples or as power/angle combinations in a format similar to
+    those recorded by EK60 echosounders (``encode_mode="power"``).
+
+    The current calibration implemented for EK80 broadband complex data
+    uses band-integrated Sv with the gain computed at the center frequency
+    of the transmit signal.
+
+    Parameters
+    ----------
```

Comment:
Add `env_params` and `cal_params`, right? https://echopype.readthedocs.io/en/stable/process.html#calibration-parameters

Reply:
actually the current text here came from there with some modifications. I think an overhaul is in order!!

Reply:
Overhaul of the doc page? If so, can you open a new issue with the specifics?

Reply:
Not the doc pages, just overhaul of the `compute_range` docstring.